test-resources/src/dequeue/tests/dequeueHandler.test.ts

```diff
@@ -15,8 +15,10 @@ import { MessageName, registeredLogs } from "../registeredLogs";
 import {
   eventNameMissingSQSRecord,
   invalidBodySQSRecord,
-  passingSQSRecord,
+  passingSQSRecordKnownSessionId,
+  passingSQSRecordUnknownSessionId,
   putItemInputForPassingSQSRecord,
+  putItemInputForPassingSQSRecordWithoutSessionId,
```

Comment:
nit: consistency in naming

```suggestion
  putItemInputForPassingSQSRecordUnknownSessionId,
```

Reply:
Updated